src/modules/job-post/job-post.service.ts

```diff
@@ -95,6 +96,47 @@ export class JobPostService {
       .getMany();
   }
 
+  async findAndFilterAll(queryParams: searchJobFiltersDto) {
+    const { category, skills, englishLevel, duration, durationType, rate } =
+      queryParams;
+    const take = queryParams.take || 100;
```

Comment:
no magic number, move to constant and make appropriate name

Reply:
fixed